src/pybind/mgr/dashboard/frontend/src/app/core/navigation/breadcrumbs/breadcrumbs.component.ts

```diff
@@ -138,4 +141,13 @@ export class BreadcrumbsComponent implements OnDestroy {
 
     return of(value as T);
   }
+
+  private getTitleFromCrumbs(crumbs: IBreadcrumb[]): string {
+    const currentLocation = crumbs
+      .map((crumb: IBreadcrumb) => {
+        return crumb.text.startsWith('http') ? '' : crumb.text;
+      })
+      .join(' > ');
+    return `Ceph${currentLocation.length > 0 ? `: ${currentLocation}` : ''}`;
```

Comment:
While this is impeccable from a functional code, nesting text interpolation is harder to read. Why not making it more explicit (the following is heavily influenced by Python stricter syntax)?:

```suggestion
    if (currentLocation.length > 0) {
        return `Ceph: ${currentLocation}`;
    } else {
        return 'Ceph';    
    }
```

Reply:
Nesting text interpolation does make it harder to read. Will update the code for better readability. Thanks!